crates/bevy_winit/src/lib.rs

```diff
@@ -148,6 +148,8 @@ pub fn winit_runner(mut app: App) {
         &mut create_window_event_reader,
     );
 
+    app.initialize();
```

Comment:
Is there a reason this is down here instead of at the top? This would be a slight change in init behavior, this now does window creation / event loop creation before running the startup systems.

Reply:
I wanted to do it right before entering the main loop.

- preparation
- initialize()
- loop {}

Reply:
The point of this change is to have access to resources created by runner in startup systems.

Reply:
Sounds reasonable to me!